assets/src/scripts/monitoring-location/components/hydrograph/date-controls.js

```diff
@@ -260,10 +260,11 @@ export const drawDateRangeControls = function(elem, store, siteno) {
             .attr('type', 'text')
             .attr('aria-describedby', 'custom-end-date-label custom-end-date-hint');
 
-        // required to init the USWDS date picker after page load
-        components.datePicker.init(elem.node());
+        // required to init the USWDS date picker after page load before calling the
+        // dateRangePicker on function
+        components.datePicker.init(dateRangePicker.node());
```

Comment:
I had to use "init" for the datePicker part of the dateRangePicker initialization. Not sure why. I reported to USWDS and others do not have the same issue and can use "on"